languages/emacs-lisp/reference/README.md

```diff
@@ -0,0 +1,39 @@
+# Emacs Lisp reference
+
+## Concepts
+
+The Emacs Lisp concept exercises are based on concepts.
+The list below contains the concepts that have been identified for the Emacs Lisp language.
+
+See [progression.md](progression.md) for planning the progression order of the Emacs Lisp track.
+
+The core features a Emacs Lisp developer should know about are:
+
+- Special forms
+- Atoms
+- Quote
+- S-expressions
+- [Evaluation][evaluation]
+- [Functions][functions]
+- [Variables][variables]
+- [Macros][macros]
+- Side-effects
+- Arguments
```

Comment:
what do you mean here precisely? the arguments we pass to a function? it's weird that we do not have them on the toplevel list of concepts

Reply:
Yeah that should be part of functions which I believe is part of lists. I'll fix the order. Arguments, meaning every item that is passed to the function yes.